sdks/java/core/src/main/java/org/apache/beam/sdk/Pipeline.java

```diff
@@ -240,6 +242,35 @@ public void setCoderRegistry(CoderRegistry coderRegistry) {
      * visited.
      */
     public void visitValue(PValue value, TransformTreeNode producer);
+
+    /**
+     * Control enum for indicating whether or not a traversal should process the contents of
+     * a composite transform or not.
+     */
+    public enum Recurse {
```

Comment:
OK, since part of the benefit of an enum over a boolean - aside from readability - is to allow expansion to non-binary decisions, it should be something more like `CompositeBehavior` or some such.